src/pages/admin/student/edit/page.tsx

```diff
@@ -1,17 +1,58 @@
 import * as React from 'react';
-import {Link} from 'react-router';
-import {adminRouteEnums} from '../../../../common/routeEnums/admin';
-
-export class EditStudentPage extends React.Component<{}, {}> {
-   public render() {
-       return (
-         <div>
-           <span> Edit Student Page: </span>
-           <br/>
-           <br/>
-           <Link to={adminRouteEnums.student.list}>Back to student list</Link>
-           <Link to={adminRouteEnums.default}>Back to Dashboard</Link>
-         </div>
-        );
+import { Link } from 'react-router';
+import { Student } from '../../../../model/student';
+import { EditStudentComponent } from './components/editStudentComponent';
+
+interface IProps extends React.Props<EditStudentPage> {
+  student: Student;
+  getStudent: (id: number) => void;
+  saveStudent: (student: Student) => void;
+}
+
+export class EditStudentPage extends React.Component<IProps, {}> {
+  public componentDidMount() {
+    // TODO: Get the Student Id from the URL
+    this.props.getStudent(32);
   }
+
+  public render() {
+    if (this.props.student.id === -1) {
```

Comment:
As previous comment, didn't want to extend the development of this issue more than strictly needed.

I guess a InsertStudentComponent will be the best option.

Regarding the -1, as soon as id is obtained from the URL the behaviour will be check if there is an  ID in the URL to decide which component render. If you agree